src/App.tsx

```diff
@@ -1,61 +1,139 @@
+import { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUserFromId(userId: number): User | null {
+  return usersFromServer.find((person) => person.id === userId) || null;
+}
+
+export const todosWithUser: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserFromId(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosWithUser);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
If it's a boolean the name should start with `is` or `has`
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
```